plugins/about/DevMode.qml

```diff
@@ -2,8 +2,7 @@
  * This file is part of system-settings
  *
  * Copyright (C) 2014 Canonical Ltd.
- *
- * Contact: Oliver Grawert <ogra@canonical.com>
+ * Copyright (C) 2020 UBports Foundation <developers@ubports.com>
```

Comment:
This e-mail does not exist. Were you expecting something different?

Reply:
I was told to use this e-mail, long ago. So, what copyright line shold we use?